vertical-stepped-line/script.js

```diff
@@ -39,24 +39,30 @@ const line = d3
 // g is our container
 const g = svg.append('g').translate([margin.left + 10, margin.top + 10]);
 
-d3.json('data.json', function(err, dataset) {
-  dataset.forEach(d => {
-    d.year = parseTime(d.year);
-    d.value = +d.value;
-  });
+d3.json('data.json', (err, dataset) => {
+  if (err) {
+    console.log(err);
```

Comment:
This doesn't _really_ handle the error. As in, if there's an error, it will still continue on with the rest of the script (and then fail).

Best thing to do would be to put a `return` after the `console.log`